src/ui/ResultList/ResultList.ts

```diff
@@ -208,14 +205,29 @@ export class ResultList extends Component {
     layout: ComponentOptions.buildStringOption({
       defaultValue: 'list',
       required: true,
-    })
+    }),
+    /**
+     * Specifies if the table header is to be displayed when layout is 'table'
+     *
+     * Default value is `true`.
+     */
+    displayTableHeader: ComponentOptions.buildBooleanOption({ defaultValue: true, depend: 'layout' }),
```

Comment:
My bad, I had removed the effect of these options but not the actual options themselves :)